src/80211_if.c

```diff
@@ -2941,6 +2974,13 @@ static int uccp420wlan_init(void)
 		}
 	}
 
+	error=start_core(hw);
```

Comment:
Before implementing the pushed code i tried your suggestion, ie loading of the fw in the init() but that did not work, this made me think that we need  to set the BT state from the driver.